core/server/api/v0.1/decorators/urls.js

```diff
@@ -35,53 +35,57 @@ const urlsForPost = (post, options) => {
             // @NOTE: this block also decorates primary_tag/primary_author objects as they
             // are being passed by reference in tags/authors. Might be refactored into more explicit call
             // in the future, but is good enough for current use-case
-            if (relation === 'tags' && post.tags) {
-                post.tags = post.tags.map(tag => urlsForTag(tag, options));
+            if (relation === 'tags' && attrs.tags) {
+                attrs.tags = attrs.tags.map(tag => urlsForTag(tag.id, tag, options));
```

Comment:
Nested properties that didn't get 'id' in their options will just pass in `undefined`, so they won't get urls assigned. Would need more thinking if we need to support this